sklearn/calibration.py

```diff
@@ -1391,6 +1396,9 @@ def from_predictions(
         >>> y_prob = clf.predict_proba(X_test)[:, 1]
         >>> disp = CalibrationDisplay.from_predictions(y_test, y_prob)
         >>> plt.show()
+
+        For another example of usage, see:
```

Comment:
This is the bit that is causing the error.

Reply:
Upon reading the documentation for doctest, I see that empty responses are not allowed. Also, a single empty line tells doctest the end of the string to test. Therefore, if there is no answer, we have to tell this to doctest. Alternatively, we can tell doctest to `SKIP` the test (but still build for documentation), or accept all responses, with `ELLIPSIS`. I chose to tell doctest to skip. I do not believe this compromises the tests, since doctest is testing in many other places, and the main output of the doctest `>>> plt.show()` is to include the graph in the documentation.